src/components/FeatureViews/Registration/RegistrationCart/PaymentBoard.js

```diff
@@ -213,21 +296,23 @@ export default function PaymentBoard() {
 	const handlePayment = async () => {
 		const paymentMethod = paymentMethodState.find(({checked}) => checked)?.value;
 		const {data: {enrollments}} = enrollmentResponse;
+		const isSameEnrollment = ({enrollment, course, student}) =>
```

Comment:
Instead of making the parameter a single object, you could simply have 3 different parameters.

Reply:
this is a trick - it allows you to give a function all of your parameters such that order doesn't matter. so if i call: `isSameEnrollment({ newEnrollment, course, student})` I wouldn't get a bug if I did `isSameEnrollment({ course, newEnrollment, student})`

Only started doing this because i spent an hour hunting down a bug where i switched up the parameter order -_-